finance/sc-im/Portfile

```diff
@@ -0,0 +1,27 @@
+PortSystem          1.0
+name                sc-im
+version             0.8.2
+license             BSD
+maintainers         nkorth
+
+categories          finance
+description         Spreadsheet Calculator Improvised
+long_description    An ncurses spreadsheet program for terminal
+
+platforms           darwin
+
+homepage            https://github.com/andmarti1424/sc-im
+master_sites        https://github.com/andmarti1424/sc-im/archive
+distfiles           v${version}.tar.gz
+
+checksums           rmd160  afabf063feda1800fd07e6ee13b4036c7d5a200f \
+                    sha256  7f00c98601e7f7709431fb4cbb83707c87016a3b015d48e5a7c2f018eff4b7f7 \
+                    size    1619617
+
+depends_build       bin:bison:bison
+depends_lib         port:ncurses
```

Comment:
upstream lists a collection of optional dependencies - to ensure [ReproducibleBuilds](https://trac.macports.org/wiki/ReproducibleBuilds) one should either add the dependency and always use it, or tell the build system not to use it even if the user would have it installed.

I haven't look how the software looks for it, if things are getting linked if the packages are present, and/or how to avoid that. But since `pkgconfig` is listed by upstream a dependency (which you might need here then as a `depends_build` as well?) it's likely using that to find the software.

Reply:
Ok, I added the optional dependencies that I think make sense to include. I'm hesitant to make it depend on gnuplot because of its long dependency chain, and because it's not really a dependency (see the comment I added to the Portfile)